.gitignore

```diff
@@ -14,3 +14,18 @@ control_vrt
 sigmf_to_vrt
 .DS_Store
 build/*
+.cproject
+.project
+.settings/
+CMakeCache.txt
+CMakeFiles/*
+cmake_install.cmake
```

Comment:
I think these don't belong here. Good practice is to build in a subdirectory `build/` within the source dir. The build directory is already ignored.